content/02.main-text.md

```diff
@@ -3,27 +3,27 @@
+In May 2017, the project, code-named the Deep Review, released a complete review as a preprint titled "Opportunities and obstacles for deep learning in biology and medicine" [@doi:10.1101/142760].
 While the article was under review, we continued to maintain the project and accepted new contributions.
-In February 2018, the article was accepted by the journal that it was initially submitted to.
+In February 2018, the article was accepted by the journal that it was initially submitted to and subsequently published in April [@doi:10.1098/rsif.2017.0387].
 We'll discuss our experience with our open review including the pros and cons of open collaborative reviews, as well as the infrastructure we created, termed Manubot, to enable open manuscript writing online.
 We also discuss the extent to which such efforts can contribute to a living and frequently updated literature.
 `TODO: discuss stage 2 maintainers`
 
 We initiated our open online review article by creating a GitHub repository (https://github.com/greenelab/deep-review) to coordinate and manage contributions.
 GitHub is a platform designed for collaborative software development that's adaptable for collaborative writing.
-From the start, we made the GitHub repository public, applying a Creative Commons Attribution [License](https://github.com/greenelab/deep-review/blob/master/LICENSE.md) to the manuscript.
+From the start, we made the GitHub repository public, applying a [Creative Commons Attribution License](https://github.com/greenelab/deep-review/blob/master/LICENSE.md) to the manuscript.
 Next, we encouraged anyone interested to contribute by proposing changes or additions.
 Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media, deciding without solicitation to contribute.
-In total, the project that we termed the "Deep Review" attracted {{deep_review_authors}} authors from 20 different institutions (and counting) who were not determined in advance.
+In total, the Deep Review attracted {{deep_review_authors}} authors from 20 different institutions (and counting) who were not determined in advance.
 
 Writing review articles in a public forum allows review authors to engage with the original researchers to clarify their methods and results and present them accurately, as exemplified [here](https://github.com/greenelab/deep-review/issues/213).
 `TODO: need archival issue link`
-In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review `TODO: define this or provide a reference?`, incentivizing the reviews with potential manuscript authorship.
+In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review [@doi:10.7554/eLife.38532], incentivizing the reviews with potential manuscript authorship.
```

Comment:
What does "incentivizing the reviews with potential manuscript authorship" mean? Should we say "incentivizing peer review of existing literature with Deep Review authorship" or similar?

Reply:
Yes, you have the correct sentiment.  I suggest keeping this paragraph about writing open review manuscripts in general.  "incentivizing peer review of existing literature with review manuscript authorship" is what I have in mind, but the repeated "review" is clunky.